doc/build-windows.md

```diff
@@ -129,6 +129,17 @@ Then build using:
 
 For further documentation on the depends system see [README.md](../depends/README.md) in the depends directory.
 
+## Install NSIS
```

Comment:
Can you change the title to be something like "Create Installer", and use the dash title method i.e ------ below the words (the same as all the rest of the titles in this document).

Reply:
well the `depends system` above it also uses `##`, it's inconsistent with the rest of the document that's true

Reply:
I guess I see @fanquake 's logic. Done.